doc/nrf/config_and_build.rst

```diff
@@ -4,9 +4,24 @@
 Configuration and building
 ##########################
 
-The |NCS| build and configuration system is based on the one from Zephyr, with some additions.
+The |NCS| build and configuration system is based on the one from :ref:`Zephyr project <zephyr:getting_started>`, with some additions.
 
-The guides in this section provide universal build and configuration steps valid for :ref:`all supported boards <app_boards>`.
+The figure below visualizes the tools and configuration methods in the |NCS|.
+All of them have a role in the creation of an application, from configuring the libraries or applications to building them.
+
+.. figure:: images/ncs-toolchain.svg
+   :alt: nRF Connect SDK tools and configuration
+
+   |NCS| tools and configuration methods
+
+* CMake generates build files based on the provided :file:`CMakeLists.txt` files, which use information from Kconfig and devicetree.
+  See the `CMake documentation`_.
+* :ref:`Devicetree <zephyr:dt-guide>` describes the hardware.
+* :ref:`zephyr:kconfig` generates definitions that configure the software.
+* Ninja (comparable to make) uses the build files to build the program, see the `Ninja documentation`_.
```

Comment:
```suggestion
* Ninja (comparable to Make) uses the build files to build the program, see the `Ninja documentation`_.
```

Does "make" need to be capitalized? I'm just asking in case it could be confused with the verb "make".

Reply:
I see both on the interwebs, so I think we can capitalize to make the reference more obvious.